examples/advanced/room_bot.py

```diff
@@ -0,0 +1,226 @@
+import asyncio
+import re
+import time
+from datetime import datetime
+from typing import List
+
+from apscheduler.schedulers.asyncio import AsyncIOScheduler
+
+from wechaty import Contact, Room, Wechaty, get_logger, Message
+
+welcome = """=============== Powered by Wechaty ===============
+-------- https://github.com/Chatie/wechaty --------
+Hello,
+I'm a Wechaty Botie with the following super powers:
+1. Find a room
+2. Add people to room
+3. Del people from room
+4. Change room topic
+5. Monitor room events
+6. etc...
+If you send a message of magic word 'ding',
+you will get a invitation to join my own room!
+__________________________________________________
+Hope you like it, and you are very welcome to
+upgrade me for more super powers!
+Please wait... I'm trying to login in..."""
+HELPER_CONTACT_NAME = '李卓桓'
+
+print(welcome)
+log = get_logger('RoomBot')
+
+
+async def checkRoomJoin(bot, room, inviteeList, inviter):
```

Comment:
Refer to the [function and function](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names) name style, we should keep the name of function and variable lowercase.